examples/distro/smoke-tests/src/test/java/com/example/javaagent/smoketest/SmokeTest.java

```diff
@@ -82,6 +82,9 @@ void startTarget(int jdk) {
             .withEnv("JAVA_TOOL_OPTIONS", "-javaagent:/opentelemetry-javaagent.jar")
             .withEnv("OTEL_BSP_MAX_EXPORT_BATCH", "1")
             .withEnv("OTEL_BSP_SCHEDULE_DELAY", "10")
+            // TODO (heya) update smoke tests to run using http/protobuf
+            // in the meantime, force smoke tests to use grpc protocol for all exporters
+            .withEnv("OTEL_EXPORTER_OTLP_PROTOCOL", "grpc")
```

Comment:
@laurit @mateuszrzeszutek any good ideas how to test `http/protobuf`? looks like our existing infra is pretty tied to armeria/grpc

Reply:
Not sure if we want to take in a new dependency, but I've had success with using https://github.com/grafana/oats/tree/main/yaml

Reply:
we could configure a collector that forwards to the test grpc endpoint

Reply:
I think we'll need to add http endpoints to the `fake-backend` -- right now it pretty much hardcodes gRPC, and it'd be ideal if it supported both.

Reply:
is it ok on a diff PR? I need to figure out how. thanks @mateuszrzeszutek.

Reply:
Yeah, I think so -- as long as w don't lose that. Please create an issue for that so we don't lose track of it

Reply:
done https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/10170